src/assets/styles/_shared.scss

```diff
@@ -440,3 +440,18 @@ h3 {
     margin: 20px 0;
   }
 }
+
+.main-logo {
+  background-image: url('/assets/img/logos/Verdance-logo-linen-black.png');
+  background-position-y: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+
+  &:hover {
+    cursor: pointer;
+    // background-color: skyblue;
+    // color: transparent; 
+    // text-shadow: 0 0 3px rgba(0,0,0,0.5); 
+    background-image: url('/assets/img/logos/Verdance-logo-pixelated-black.png');
```

Comment:
browsers might not load this `.main-logo:hover` image right away, causing delay to show pixelate effect.
There are probably a number of ways to achieve desired behavior. If you request the `<img>` w/ `display=none` request should still be made and could be displayed or the cache version of the image could be used in the `background-image url()`

Reply:
you can also use the `preload` option within `link`: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/preload

Reply:
Cool, I didn't know about `preload`!

I couldn't get rid of the blurriness regardless of using `<img>` or `background-image: url()` for the Verdance logo text. See below.

![Screenshot 2024-07-08 at 12 58 42 PM](https://github.com/VerdanceTeam/verdance-site/assets/17188013/a13ee8c5-4928-4db4-8228-369425c4a321)

In order to work around this, I left the main logo text in `<span>Verdance</span>`, but changed visibility + background-image for hover